packages/ui-library/src/components/form-caption/index.stories.ts

```diff
@@ -44,12 +45,20 @@ export default {
       },
     },
 
+    showIcon: {
+      description: 'Choose if component has an icon.',
+      defaultValue: true,
+      control: { type: 'boolean' },
+      table: { category: 'Content / Settings' },
+    },
+
     icon: {
       options: [undefined, ...PureIconKeys],
       control: { type: 'select' },
+      defaultValue: 'blr360',
       description: 'Select an icon which is displayed inside of the input.',
       table: {
-        category: 'Content / Setting',
+        category: 'Content / Settings',
```

Comment:
You fixed an issue I did not even see at first here. "Content / Settings" is the correct name for the category. Could you also fix it for the message prop, so message, and icon are in the same category. Please also check, that the order is correct. First message, then icon.